migration/i8.2z/oracle/202102201441_IDEMPIERE-4709.sql

```diff
@@ -0,0 +1,60 @@
+SET SQLBLANKLINES ON
+SET DEFINE OFF
+
+-- Feb 20, 2021, 1:45:25 PM BRT
+INSERT INTO AD_Ref_List (AD_Ref_List_ID,Name,AD_Reference_ID,Value,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,EntityType,AD_Ref_List_UU) VALUES (200504,'Script',255,'S',0,0,'Y',TO_DATE('2021-02-20 13:45:25','YYYY-MM-DD HH24:MI:SS'),100,TO_DATE('2021-02-20 13:45:25','YYYY-MM-DD HH24:MI:SS'),100,'D','f3acb050-7407-481a-9177-022c134cf3a5')
+;
+
+-- Feb 20, 2021, 1:46:44 PM BRT
+INSERT INTO AD_Column (AD_Column_ID,Version,Name,Description,Help,AD_Table_ID,ColumnName,FieldLength,IsKey,IsParent,IsMandatory,IsTranslated,IsIdentifier,SeqNo,IsEncrypted,AD_Reference_ID,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,AD_Element_ID,IsUpdateable,IsSelectionColumn,EntityType,IsSyncDatabase,IsAlwaysUpdateable,IsAutocomplete,IsAllowLogging,AD_Column_UU,IsAllowCopy,SeqNoSelection,IsToolbarButton,IsSecure,IsHtml) VALUES (214413,0,'Script','Dynamic Java Language Script to calculate result','Use Java language constructs to define the result of the calculation',489,'Script',2000,'N','N','N','N','N',0,'N',14,0,0,'Y',TO_DATE('2021-02-20 13:46:44','YYYY-MM-DD HH24:MI:SS'),100,TO_DATE('2021-02-20 13:46:44','YYYY-MM-DD HH24:MI:SS'),100,1718,'Y','N','D','N','N','N','Y','d0873a87-64d0-4d21-bf51-5b5b6ce1645a','Y',0,'N','N','N')
+;
+
+-- Feb 20, 2021, 1:46:46 PM BRT
+ALTER TABLE AD_PrintFormatItem ADD Script VARCHAR2(2000 CHAR) DEFAULT NULL 
+;
+
+-- Feb 20, 2021, 1:47:32 PM BRT
+INSERT INTO AD_Field (AD_Field_ID,Name,Description,Help,AD_Tab_ID,AD_Column_ID,IsDisplayed,DisplayLength,SeqNo,IsSameLine,IsHeading,IsFieldOnly,IsEncrypted,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,IsReadOnly,IsCentrallyMaintained,EntityType,AD_Field_UU,IsDisplayedGrid,SeqNoGrid,ColumnSpan,NumLines) VALUES (206597,'Script','Dynamic Java Language Script to calculate result','Use Java language constructs to define the result of the calculation',428,214413,'Y',2000,10,'N','N','N','N',0,0,'Y',TO_DATE('2021-02-20 13:47:32','YYYY-MM-DD HH24:MI:SS'),100,TO_DATE('2021-02-20 13:47:32','YYYY-MM-DD HH24:MI:SS'),100,'N','Y','D','a969e724-453a-4c2a-ad37-628168e6425d','Y',10,5,3)
+;
+
+-- Feb 20, 2021, 1:49:00 PM BRT
+UPDATE AD_Field SET DisplayLogic='@PrintFormatType@=''S''', AD_Reference_Value_ID=NULL, AD_Val_Rule_ID=NULL, IsToolbarButton=NULL,Updated=TO_DATE('2021-02-20 13:49:00','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Field_ID=206597
+;
+
+-- Feb 20, 2021, 1:51:25 PM BRT
+DELETE  FROM  AD_Field_Trl WHERE AD_Field_ID=206597
+;
+
+-- Feb 20, 2021, 1:51:25 PM BRT
+DELETE FROM AD_Field WHERE AD_Field_ID=206597
```

Comment:
It seems the insert, update and 2 delete here for AD_Field_ID=206597 is a mistake that have been captured into the migration script.

Reply:
Removed From Migration scripts.